app/_src/gateway-operator/production/upgrade/data-plane/blue-green.md

```diff
@@ -0,0 +1,173 @@
+    ```
+
+1. Test it out by patching the `DataPlane` with a new `image`:
+
+    ```bash
+    $ kubectl patch dataplane bluegreen --type='json' -p='[{"op": "replace", "path": "/spec/deployment/podTemplateSpec/spec/containers/0/image", "value":"kong:3.3.1"}]'
+    dataplane.gateway-operator.konghq.com/bluegreen patched
+    ```
+
+    After this patch gets applied you'll be able to access the new {{ site.base_gateway }} `Pod`s via the "preview" ingress `Service`.
+
+1. To find the "preview" `Service` you can look up `DataPlane` status, and more specifically its `rollout` field:
+
+    ```bash
+    $ kubectl get dataplane bluegreen -o jsonpath-as-json='{.status.rollout}
+    [
+        {
+            "conditions": [
+                {
+                    "lastTransitionTime": "2023-09-21T11:40:25Z",
+                    "message": "",
+                    "observedGeneration": 2,
+                    "reason": "AwaitingPromotion",
+                    "status": "False",
+                    "type": "RolledOut"
+                }
+            ],
+            "deployment": {
+                "selector": "6cf0d993-2319-43d5-bfdc-e2cadd6bd7e3"
+            },
+            "services": {
+                "adminAPI": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "None"
+                        }
+                    ],
+                    "name": "dataplane-admin-bluegreen-cx6nq"
+                },
+                "ingress": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateLoadBalancer",
+                            "type": "IPAddress",
+                            "value": "172.18.0.101"
+                        },
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "10.96.28.2"
+                        }
+                    ],
+                    "name": "dataplane-ingress-bluegreen-2249g"
+                }
+            }
+        }
+    ]
+    ```
+
+    Here you can see the ingress `Service` name that was created for you to validate the new set of `Pod`s.
+    Its name is stored in `status.rollout.service`.
+    For convenience its addresses (together with their `type`s and `sourceType`s) are stored `status.rollout.services[].ingress`.
+
+    Notice that `status.rollout.condtition` contains a condition with Type `RolledOut` and `Reason` set to `AwaitingPromotion`.
+    This means that everything is ready to promote the `DataPlane` with staged changes.
+    Before we do so, let's test it.
+
+1. You can access the spawned "preview" `Service` by using its LB address (taken from the `status.rollout.services.ingress` field):
+
+    ```bash
+    $ curl -s -D - -o /dev/null 172.18.0.101
```

Comment:
https://datatracker.ietf.org/doc/html/rfc5737 use the IP address reserved for documentation